apps/web/screens/Syslumenn/SubPage.tsx

```diff
@@ -0,0 +1,217 @@
+  subpage: Query['getOrganizationSubpage']
+  namespace: Query['getNamespace']
+}
+
+const SubPage: Screen<SubPageProps> = ({
+  organizationPage,
+  subpage,
+  namespace,
+}) => {
+  const n = useNamespace(namespace)
+  const { linkResolver } = useLinkResolver()
+
+  const navList: NavigationItem[] = organizationPage.menuLinks.map(
+    ({ primaryLink, childrenLinks }) => ({
+      title: primaryLink.text,
+      href: primaryLink.url,
+      active:
+        subpage.menuItem.url === primaryLink.url ||
+        childrenLinks.some((link) => link.url === subpage.menuItem.url),
+      items: childrenLinks.map(({ text, url }) => ({
+        title: text,
+        href: url,
+        active: url === subpage.menuItem.url,
+      })),
+    }),
+  )
+
+  console.log(subpage.slices)
+
+  return (
+    <OrganizationWrapper
+      pageTitle={subpage.title}
+      pageDescription={subpage.description}
+      organizationPage={organizationPage}
+      pageFeaturedImage={subpage.featuredImage}
+      fullWidthContent={true}
+      breadcrumbItems={[
+        {
+          title: 'Ísland.is',
+          href: linkResolver('homepage').as,
+        },
+        {
+          title: n('organizations', 'Stofnanir'),
+          href: linkResolver('organizations').as,
+        },
+        {
+          title: organizationPage.title,
+          href: linkResolver('organizationpage', [organizationPage.slug]).as,
+        },
+      ]}
+      navigationData={{
+        title: n('navigationTitle', 'Efnisyfirlit'),
+        items: navList,
+        titleLink: {
+          href: linkResolver('organizationpage', [organizationPage.slug]).as,
+          active: false,
+        },
+      }}
+    >
+      <GridContainer>
+        <Box
+          className={styles.intro}
+          paddingTop={[4, 4, 0]}
+          paddingBottom={[4, 4, 6]}
+        >
+          <GridRow>
+            <GridColumn
+              span={['12/12', '12/12', subpage.links.length ? '7/12' : '12/12']}
+            >
+              <h2 className={styles.heading}>{subpage.title}</h2>
+            </GridColumn>
+          </GridRow>
+          <GridRow>
+            <GridColumn
+              span={['12/12', '12/12', subpage.links.length ? '7/12' : '12/12']}
+            >
+              <div
+                className={
+                  subpage.slices.length
+                    ? styles.description
+                    : styles.smallDescription
+                }
+              >
+                <Markdown>{subpage.description}</Markdown>
+              </div>
+            </GridColumn>
+            {subpage.links.length > 0 && (
+              <GridColumn
+                span={['12/12', '12/12', '4/12']}
+                offset={[null, null, '1/12']}
+              >
+                <ul>
+                  {subpage.links.map((link) => (
+                    <li>
+                      <Link href={link.url}>
+                        <a className={styles.link} href={link.url}>
```

Comment:
the `Link` component will render an anchor for you, so no need for the duplication here, you should use the `Text` component for styling the link